greatestPalindrome.js

```diff
@@ -0,0 +1,57 @@
+
+/*Given a string return the longest palindrome that can be constructed by removing or shuffling characters. 
+Example: 
+'aha' -> 'aha' 
+'ttaatta' -> ' ttaaatt' 
+'abc' -> 'a' or 'b' or 'c' 
+'gggaaa' -> 'gaaag' or 'aggga' 
+Note if there are multiple correct answers you only need to return 1 palindrome.
+*/
+
+//tests
+dictionaryCreator(word)
```

Comment:
like i said before all of this function calling and logs should be at the end of the file